src/Orders/WebhookCallOrderHistoryTest.php

```diff
@@ -0,0 +1,211 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Craftzing\Laravel\MollieWebhooks\Orders;
+
+use Craftzing\Laravel\MollieWebhooks\Refunds\RefundId;
+use Craftzing\Laravel\MollieWebhooks\Testing\Doubles\FakeMollieWebhookCall;
+use Craftzing\Laravel\MollieWebhooks\Testing\IntegrationTestCase;
+use Generator;
+use Mollie\Api\Types\RefundStatus;
+
+use function array_merge;
+use function json_encode;
+
+final class WebhookCallOrderHistoryTest extends IntegrationTestCase
```

Comment:
Same here: replace the `payment` reference with `order` in all comments and generator keys.